AsyncDisplayKit/Layout/ASStaticLayoutSpec.mm

```diff
@@ -38,10 +39,8 @@ - (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize
 {
   CGSize maxConstrainedSize = CGSizeMake(constrainedSize.max.width, constrainedSize.max.height);
   
-  NSArray *children = self.children;
-  NSMutableArray *sublayouts = [NSMutableArray arrayWithCapacity:children.count];
-
-  for (id<ASLayoutable> child in children) {
+  NSMutableArray *sublayouts = [NSMutableArray array];
+  for (id<ASLayoutable> child in self) {
```

Comment:
Hmm this contradicts that http://nshipster.com/benchmarking/ . On checking the disassembly, it's true that the capacity is discarded but it's discarded fairly deep inside the code, possibly as a last-minute change. I believe we should still provide it for two reasons:

- Future implementations of Foundation might use the capacity.
- NSHipster benchmark claims it's faster which makes sense because `-init` calls through to `-initWithCapacity`